src/client/common/application/types.ts

```diff
@@ -1599,77 +1594,6 @@ export interface IVSCodeNotebook {
     registerNotebookKernelProvider(selector: NotebookDocumentFilter, provider: NotebookKernelProvider): Disposable;
 }
 
-export const IAuthenticationService = Symbol('IAuthenticationService');
```

Comment:
Not required anymore, we use the stable VS Code API `SecretsStorage`

Reply:
Didn't you already fix this in another PR?